.devcontainer/certs/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.cnf
```

Comment:
curious why don't we directly ignore the whole folder? it's never impossible we don't have other cert files under the directory, right?